### PR TITLE
Fix: text overflowing in bookmarks

### PR DESCRIPTION
**Pull request by yordis**
## Proposed change

- Fixed the text-overflow in bookmarks (see screenshots)

### Before

Screenshot 2024-10-29 at 8 32 06 PM

### After

Screenshot 2024-10-29 at 8 46 12 PM

## Type of change

- [ ] New service widget
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation only
- [ ] Other (please explain)

## Checklist:

- [x] If applicable, I have added corresponding documentation changes.
- [x] If applicable, I have reviewed the [feature](https://gethomepage.dev/more/development/#new-feature-guidelines) and / or [service widget guidelines](https://gethomepage.dev/more/development/#service-widget-guidelines).
- [x] I have checked that all code style checks pass using [pre-commit hooks](https://gethomepage.dev/more/development/#code-formatting-with-pre-commit-hooks) and [linting checks](https://gethomepage.dev/more/development/#code-linting).
- [x] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.

### CODE REVIEW

File: package.json
Comment (shamoon) [inline package.json:1]: Please revert this change

Comment (yordis) [reply package.json:1]: Aye! Doing it, hold on, corepack will continue adding the damn thingy back!

**Review by shamoon — APPROVED** [review]
Thanks, one fix:


Screenshot 2024-10-29 at 9 15 49 PM

(before:)

Screenshot 2024-10-29 at 9 15 54 PM
